package.json

```diff
@@ -91,6 +91,15 @@
     "yargs": "^17.7.2"
   },
   "dependencies": {
+    "@modelcontextprotocol/sdk": "^1.17.0",
```

Comment:
These should be updated in the core package, not here

Reply:
Great point:

  - Dependencies are now updated only in individual packages where they're used:
    - @modelcontextprotocol/sdk: ^1.11.0 → ^1.17.0 in packages/core/package.json
    - @modelcontextprotocol/sdk: ^1.15.1 → ^1.17.0 in packages/cli/package.json
    - @modelcontextprotocol/sdk: ^1.15.1 → ^1.17.0 in packages/vscode-ide-companion/package.json
    - google-auth-library: ^9.11.0 → ^10.2.0 in packages/core/package.json
  - NO dependencies added to root package.json